cmd/antrea-controller/options_test.go

```diff
@@ -0,0 +1,178 @@
+		},
+		{
+			name: "invalid ClusterCIDRs",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0", "a:b::0/64"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 70,
+			},
+			expectedErr: "cluster CIDRs",
+		},
+		{
+			name: "invalid ClusterCIDRs num",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 780,
+			},
+			expectedErr: "at least one cluster CIDR must be specified",
+		},
+		{
+			name: "invalid ClusterCIDRs num",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64", "20.20.0.0/24"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "at most two cluster CIDRs may be specified",
+		},
+		{
+			name: "at most one cluster CIDR of each type",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "20.20.0.0/24"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "at most one cluster CIDR may be specified for each IP family",
+		},
+		{
+			// 	ipamIPv4MaskLo = 16  ipamIPv4MaskHi      = 30
+			name: "invalid node IPv4 CIDR mask size",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 15,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "node IPv4 CIDR mask size",
+		},
+		{
+			//	ipamIPv6MaskLo = 64  ipamIPv6MaskHi = 126
+			name: "invalid node IPv6 CIDR mask size",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 63,
+			},
+			expectedErr: "node IPv6 CIDR mask size",
+		},
+		{
+			name: "invalid service CIDR",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64"},
+				ServiceCIDR:          "1",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "service CIDR",
```

Comment:
ditto, the error is not long enough to exclude other errors when you are using `ErrorContains` to check the result.